### PR TITLE
Document how to retrieve mapped user directory attributes in Java

### PR DESCRIPTION
**Pull request by thomasdelhomenie**

### CODE REVIEW

**Review by boubaker — APPROVED** [review]

File: docs/LDAP.rst [hunk 1]
```diff
@@ -372,7 +372,21 @@ you must add this configuration snippet under the “attributes” tag in the fi
 				   </attribute>
 		...
 			   </attributes>
-	
+
+These attributes can be retrieved in the User Profile with the Java API:
```
Comment (rdenarie) [inline docs/LDAP.rst]:
```suggestion
These attributes can be retrieved in the Gatein User Profile with the Java API:
```

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: docs/LDAP.rst (hunk 1 above)
Comment (boubaker) [reply docs/LDAP.rst]: Why adding **Gatein** (the product name, that only internal eXoers should know) ? If you want to be more specific, you should use *IDM* instead IMO.

Comment (rdenarie) [reply docs/LDAP.rst]: You're right.
I mean Portal User Profile, to make difference with Social User Profile
```suggestion
These attributes can be retrieved in the Portal User Profile with the Java API:
```

Comment (thomasdelhomenie) [reply docs/LDAP.rst]: I agree "Portal" is better than "Gatein". @boubaker ok for you for "Portal" ?

Comment (boubaker) [reply docs/LDAP.rst]: I let you decide

Comment (thomasdelhomenie) [reply docs/LDAP.rst]: ok, let's go for Portal.

**Review by rdenarie — APPROVED** [review]
LGTM

**Pull request merged**